src/main/java/org/isf/stat/manager/JasperReportsManager.java

```diff
@@ -1097,4 +1099,37 @@ private static Date toDate(LocalDate localDate) {
 		return Optional.ofNullable(localDate).map(t -> Date.from(t.atStartOfDay().atZone(ZoneId.systemDefault()).toInstant())).orElse(null);
 	}
 
-}
+	public JasperReportResultDto getInventoryReportPdf(MedicalInventory inventory, String jasperFileName, int printQtyReal) throws OHServiceException {
+		try {
+			HashMap<String, Object> parameters = new HashMap<>();
+			Hospital hosp = hospitalManager.getHospital();
+			String status = MessageBundle.getMessage(inventory.getStatus());
+			parameters.put("Hospital", hosp.getDescription());
+			parameters.put("Address", hosp.getAddress());
+			parameters.put("City", hosp.getCity());
+			parameters.put("Email", hosp.getEmail());
+			parameters.put("Telephone", hosp.getTelephone());
+			parameters.put("inventoryId", String.valueOf(inventory.getId()));
+			parameters.put("inventoryReference", inventory.getInventoryReference());
+			parameters.put("inventoryStatus", status != null ? status : inventory.getStatus());
+			parameters.put("printQtyReal", printQtyReal);
+
+			LocalDateTime inventoryDateTime = inventory.getInventoryDate();
+			DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
```

Comment:
Why did you not use the static variable defined at line 89?

Reply:
I have not make the remark for this variable declaration thanks.